Tests/TestingApps/PaywallsTester/PaywallsTester/Config/SamplePaywalls.swift

```diff
@@ -633,25 +633,335 @@ private extension SamplePaywallLoader {
+        .text(.init(
+            text: "Title Text",
+            textLid: "package_title",
+            fontFamily: "",
+            fontWeight: .regular,
+            color: .init(light: "#000000"),
+            padding: .zero,
+            textStyle: .body
+        ))
+    }()
+
+    static func packagesSample(slide: Int) -> [PaywallComponent] {
+        return switch slide {
+        case 1:
+            packagesSample1
+        case 2:
+            packagesSample2
+        case 3:
+            packagesSample3
+        case 4:
+            packagesSample4
+        case 5:
+            packagesSample5
+        case 6:
+            packagesSample6
+        default:
+            [PaywallComponent]()
+        }
+    }
+
+//    static var packagesSample: [PaywallComponent] = {
+//        [.packageGroup(.init(type: .packageGroup, defaultSelectedPackageID: "none", components:
+//        [.packageGroup(.init(type: .packageGroup, defaultSelectedPackageID: "$rc_weekly", components: packageWall)),
+//         .packageGroup(.init(type: .packageGroup, defaultSelectedPackageID: "$rc_monthly", components: packageWall)),
+//         purchasePackageButton]))]
+//    }()
+
+//    static var packagesSample: [PaywallComponent] = {
+//        [.packageGroup(.init(type: .packageGroup, defaultSelectedPackageID: "$rc_weekly", components: packageWall))]
+//    }()
+
+    // SLIDE 1
+    static var packagesSample1: [PaywallComponent] = {
+        [.packageGroup(.init(type: .packageGroup, defaultSelectedPackageID: "$rc_weekly", components: [purchasePackageButton]))]
+    }()
+
+    // SLIDE 2
+    static var packagesSample2: [PaywallComponent] = {
+        [.packageGroup(.init(type: .packageGroup, defaultSelectedPackageID: "$rc_weekly", components: [purchasePackageButton]))] +
+        [.packageGroup(.init(type: .packageGroup, defaultSelectedPackageID: "$rc_monthy", components: [purchasePackageButton]))]
+    }()
+
+    // SLIDE 3
+    static var packagesSample3: [PaywallComponent] = {
+        [.packageGroup(.init(type: .packageGroup, defaultSelectedPackageID: "$rc_weekly", components: packageWall))]
+    }()
+
+    // SLIDE 4
+    static var packagesSample4: [PaywallComponent] = {
+        [.packageGroup(.init(type: .packageGroup, defaultSelectedPackageID: "$rc_weekly", components: packageWall)),
+         spacer,
+         .packageGroup(.init(type: .packageGroup, defaultSelectedPackageID: "$rc_monthly", components: packageWall))]
+    }()
+
+    // SLIDE 5
+    static var packagesSample5: [PaywallComponent] = {
+        [.packageGroup(.init(type: .packageGroup, defaultSelectedPackageID: "$rc_weekly", components:[textPackage1, textPackage2, purchasePackageButton]))]
+    }()
+
+    static var packagesSample6: [PaywallComponent] = {
+        [.packageGroup(.init(type: .packageGroup, defaultSelectedPackageID: "$rc_weekly", components:[packageVStackA, packageVStackB, purchasePackageButton]))]
+    }()
+
+    // SLIDE 7 - unsupported what is this even
```

Comment:
?

Reply:
That was for the nested package groups, which I decided made no sense. I'll remove this.